src/lib/mod.rs

```diff
@@ -85,3 +105,32 @@ pub fn build_lex<TokId: Copy + Eq + TryFrom<usize>>(s: &str) -> Result<LexerDef<
     parse_lex(s)
 }
 
+/// Statically compile all `.l` files in the `src` directory. A file `src/x.l` can then be used in
+/// a crate with `lrlex_mod!(x)`: it exposes a function `lexerdef()` which returns a
+/// [`LexerDef`](struct.LexerDef.html) that can then be used as normal.
+pub fn process_src_dir<TokId: Copy + fmt::Debug + Eq + TryFrom<usize> + TypeName>() -> Result<(), Box<Error>> {
+    let mut pd = current_dir()?;
+    pd.push("src");
+    build::process_dir::<TokId, _, _>(pd, var("OUT_DIR").unwrap())
+}
+
+/// Statically compile a specific `.l`. A file `/a/b/x.l` will be compiled into `/a/b/x.rs`. Note
+/// that if `/a/b` is not the `src` directory of your crate, you will have to manually `include!`
+/// the resulting `.rs` file (since [`lrlex_mod`](macro.lrlex_mod.html) only deals with `.rs` files
+/// in the `src` directory).
+pub fn process_file<TokId, P, Q>(inp: P,
+                                 outp: Q)
+                              -> Result<(), Box<Error>>
+                           where TokId: Copy + Debug + Eq + TryFrom<usize> + TypeName,
+                                 P: AsRef<Path>,
+                                 Q: AsRef<Path>
+{
+    build::process_file::<TokId, P, Q>(inp, outp)
+}
+
+#[macro_export]
```

Comment:
Should `#[macro_export]` be below the comment?

Reply:
Looking at Rust's library, it looks like it should be. Will change.